### PR TITLE
lession 1 activity

### PR DESCRIPTION
**Pull request by trtpg**
Add lesson 1 activity break_me.py file

### CODE REVIEW

**Comment by trtpg** [comment]
Add Lesson 2 exercises.

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (mdgregor) [inline .idea/vcs.xml]: you'll want to add to your .gitignore file the `.idea` and `.idea/` . See the class gitignore here: https://github.com/UWPCE-PythonCert-ClassRepos/Wi2018-Online/blob/master/.gitignore

File: students/PhongT/lesson02/fizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"""
+Lesson 02
+fizz buzz Exercise
+"""
+
+
+def printFizzBuzz():
```
Comment (mdgregor) [inline students/PhongT/lesson02/fizzBuzz.py]: underscores for naming in python, no camel case allowed :) https://www.python.org/dev/peps/pep-0008/

File: students/PhongT/lesson02/series.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    elif n == 1:
+        return 2
+    elif n == 2:
+        return 1
+    else:
+        return lucas(n-1) + lucas(n-2)
+
+
+
+def sum_series(n, f0 = 0, f1 = 1):
```
Comment (mdgregor) [inline students/PhongT/lesson02/series.py]: try to use full words as variable names whenever possible.  f0 and f1 are not descriptive even though i know what you mean.

**Comment by mdgregor** [comment]
Great job!

**Pull request merged**